README.md

```diff
@@ -688,6 +689,14 @@ Type: _string_
 
 Default: `""`
 
+#### `domain_website`
+
+Set this to the domain that you want to deploy your website to. By default it will be deployed to a *.pages.dev
```

Comment:
```suggestion
Set this to the domain that you want to deploy your website to. By default, it will be deployed to a `PROJECT_NAME.pages.dev` Cloudflare Pages URL.
```

Reply:
From your message above @anujdeshpande could someone provide a more generic input like `balena.io/docs` or `docs.balena.io` to configure both domain and BaseURL together for docusaurus. 

I know configuring docusaurus is easy but the hard part would be writing the scripts that would end up changing CNAME fields of URL entry in Cloudflare DNS. We could do the hard part manually or later but still give folks the option at least to define both URL (balena.io) + baseURL (/docs)

Example: Input can be `balena.io/cds/docs` = balena.io + /cds/docs 
This will make this input more versatile. I am asking this because I would need this for balenaCloud docs deployment: https://github.com/balena-io/docs/actions/runs/3694510530

This PR will fail because the base URL will be incorrect in docusaurus config.

Reply:
1. We can easily use a node URL library on the docusaurusify script side to parse the URL input. 
2. Figure out domain and subdomain 
3. End up configuring the correct values. 
4. Then manually configuring Cloudflare DNS ourselves.

Reply:
> @vipulgupta2048 see the comment above

@anujdeshpande Can you provide a link to which comment you are exactly referring to?

Reply:
@vipulgupta2048 https://github.com/product-os/flowzone/pull/405#issuecomment-1365349994